eap/eap71-tx-recovery-s2i.json

```diff
@@ -158,6 +165,47 @@
         }
     ],
     "objects": [
+        {
+            "apiVersion": "v1",
+            "kind": "Role",
+            "metadata": {
+                "name": "listing-pod-role"
+            },
+            "rules": [
+                {
+                    "apiGroups": null,
+                    "attributeRestrictions": null,
+                    "resources": ["pods", "pods/log"],
+                    "verbs": ["list", "get"]
+                }
+            ]
+        },
+        {
+            "apiVersion": "v1",
+            "kind": "ServiceAccount",
+            "metadata": {
+                "name": "listing-pod"
+            }
+        },
+        {
+            "apiVersion": "v1",
+            "kind": "RoleBinding",
+            "metadata": {
+                "name": "listing-pod"
+            },
+            "subjects": [
+                {
+                    "kind": "ServiceAccount",
+                    "name": "listing-pod",
+                    "namespace": "${NAMESPACE}"
```

Comment:
Is namespace required here?  If so, that might present a usability problem.  I thought most references defaulted to the containing namespace if not specified.